tests/test_nodes.py

```diff
@@ -85,6 +85,22 @@ def test_valid_inputs(type_: type, input_: Any, output_: Any) -> None:
         (IntegerNode, "-abc"),
         (BooleanNode, "Nope"),
         (BooleanNode, "Yup"),
+        (StringNode, [1, 2]),
```

Comment:
I want to see invalid assignment to AnyNode here as well (dict, list, DictConfig and ListConfig, also illegal values like objects that are not Structured Configs).

Reply:
Okay

Reply:
By assignment I mean set_value(), AnyNode can be replaced by a container.

Reply:
I'll extend the test_set_anynode with more values then.